Tangem/Modules/OnrampSettings/OnrampSettingsViewModel.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  OnrampSettingsViewModel.swift
+//  TangemApp
+//
+//  Created by Aleksei Muraveinik on 3.11.24..
+//  Copyright © 2024 Tangem AG. All rights reserved.
+//
+
+import Combine
+import TangemExpress
+
+final class OnrampSettingsViewModel: ObservableObject {
+    @Published private(set) var selectedCountry: OnrampCountry?
+
+    private weak var coordinator: OnrampSettingsRoutable?
+
+    init(repository: OnrampRepository, coordinator: OnrampSettingsRoutable) {
+        self.coordinator = coordinator
+        selectedCountry = repository.preferenceCountry
+
+        repository.preferenceCountryPublisher
+            .assign(to: &$selectedCountry)
```

Comment:
без `store(in: )` не имеет смысла же
```suggestion
        selectedCountry  = repository.preferenceCountry
```

Reply:
.assign(to: ) работает не так как обычные паблишеры, глянь эпл доку

Reply:
https://developer.apple.com/documentation/combine/publisher/assign(to:)

Reply:
хм, интересно, не знал

Reply:
не, не надо так